postgres/15/healthcheck.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# inspired by https://github.com/docker-library/healthcheck
+set -eo pipefail
+
+# check if restore is in progress (see 2_restore_dump.sh)
+if [ -f /var/lib/postgresql/restore.inProgress ]; then
+  echo "Database restore in progress. Not yet healthy."
+  exit 1
+fi
+
+user="${POSTGRES_USER:-bonita}"
+db="${POSTGRES_DB:-$user}"
+export PGPASSWORD="${POSTGRES_PASSWORD:-bpm}"
+
+args=(
+  # force postgres to not use the local unix socket (test "external" connectivity)
+  --host "$host"
+  --username "$user"
+  --dbname "$db"
+  --quiet --no-align --tuples-only
+)
+
+if select="$(echo 'SELECT 1' | psql "${args[@]}")" && [ "$select" = '1' ]; then
+  echo "${host} is healthy"
+  exit 0
+fi
+
+echo "${host} is not healthy"
```

Comment:
Each time we check that postgres is NOT healthy, we will issue this message. Is is what we want?

Reply:
Maybe not indeed.